src/services/legal-services.ts

```diff
@@ -304,4 +307,98 @@ export default class LegalServices {
         throw new Error('Invalid API response')
       })
   }
+
+  /** Fetches the document info for a business. EG. Rules / Memorandum keys
+   * @param businessId the id of the business
+   * @returns a promise to return the data
+   */
+  static async fetchBusinessDocuments (businessId: string): Promise<BusinessDocumentsIF> {
+    const url = `businesses/${businessId}/documents`
+
+    return axios.get(url)
+      .then(response => {
+        if (response?.data) {
+          return response.data
+        }
+        // eslint-disable-next-line no-console
+        console.log('fetchBusinessDocuments() error - invalid response =', response)
+        throw new Error('Invalid API response')
+      })
+  }
+
+  /**
+   * Fetches a document and prompts browser to open/save it.
+   * @param document the document info object
+   * @returns the axios response
+   */
+  static async fetchDocument (document: DocumentIF): Promise<AxiosResponse> {
```

Comment:
Straight from business-filings-ui, should go in bcrs-shared-components

Reply:
Actually, we don't have any "API services" in the shared components. We _could_, but we'd have to pass them an Axios instance. I'm OK with keeping this here unless you see a clean future way.

Reply:
Probably could create an initialize method to pass in the axios instance. 

Would need to put a check to see if it was initialized in the methods.

Reply:
So, should we move the API services to the shared components repo soon, or later? If soon then please create a ticket, otherwise we'll just try to remember.

Reply:
New ticket is bcgov/entity#16489.